script/build.js

```diff
@@ -638,7 +641,7 @@ if (options.buildtype !== 'development') {
     const manifest = {};
     Object.keys(originalManifest).forEach((originalManifestKey) => {
       const matchData = originalManifestKey.match(/(.*)\.js$/);
-      if (matchData !== null && matchData[1] !== 'vendor') {
```

Comment:
Why remove the vendor check here?

Reply:
Before the vendor file followed a different naming convention and lacked `.entry` in the file name.